websites/V/VtuberWiki/presence.ts

```diff
@@ -0,0 +1,118 @@
+type StaticPagesType = Record<string, PresenceData>;
+
+const presence = new Presence({
+		clientId: "1164189020922843257",
+		appMode: true,
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	staticPages: StaticPagesType = {
+		"/": {
+			details: "Viewing the homepage",
+			buttons: [{ label: "View Website", url: "https://wiki.hylia.dev" }],
+		},
+		"/faq": {
+			details: "Viewing the FAQ",
+			buttons: [{ label: "View Website", url: "https://wiki.hylia.dev/faq" }],
+		},
+		"/api": {
+			details: "Viewing the API",
+			buttons: [{ label: "View Website", url: "https://wiki.hylia.dev/api" }],
+		},
+	}; // Took this from websites/S/Shoob but I rewrote some of it. :heart:
+
+function fNumber(number: number): string {
+	return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+}
+
+presence.on("UpdateData", async () => {
+	const { pathname, href } = window.location,
```

Comment:
This repository prefers `document.location` rather than `window.location`
```suggestion
	const { pathname, href } = document.location,
```